packages/wrangler/package.json

```diff
@@ -159,16 +159,16 @@
 		"shell-quote": "^1.8.1",
 		"shellac": "^0.8.0",
 		"signal-exit": "^3.0.7",
-		"strip-ansi": "^7.0.1",
+		"strip-ansi": "^7.1.0",
 		"supports-color": "^9.2.2",
 		"timeago.js": "^4.0.2",
 		"ts-dedent": "^2.2.0",
 		"ts-json-schema-generator": "^1.5.0",
 		"undici": "^5.28.4",
 		"update-check": "^1.5.4",
-		"vitest": "^1.6.0",
```

Comment:
I had to bring this version down to normalize, we'll bring it back up when it works for everything

Reply:
this isn't true anymore, reverted this one.